src/ucs/stats/serialization.c

```diff
@@ -244,8 +251,20 @@ ucs_stats_serialize_binary_recurs(FILE *stream, ucs_stats_node_t *node,
     /* Name */
     ucs_stats_write_str(node->name, stream);
 
-    /* Counters */
-    ucs_stats_write_counters(node->counters, cls->num_counters, stream);
+    /* Filter output */
+    filter_node = node->filter_node;
+    filtered_counter_index = 0;
+    ucs_for_each_bit(counter_index, filter_node->counters_bitmask) {
+        ucs_stats_node_t *temp_node;
```

Comment:
can move this to line 239

Reply:
Fixed.

Reply:
pls move " ucs_stats_node_t *temp_node;" to line 239

Reply:
Fixed.